examples/complete/main.tf

```diff
@@ -1,7 +1,53 @@
-module "example" {
+provider "aws" {
+  region = var.region
+}
+
+module "vpc" {
+  source  = "cloudposse/vpc/aws"
+  version = "0.28.1"
+
+  cidr_block = "172.16.0.0/16"
+
+  context = module.this.context
+}
+
+module "subnets" {
+  source  = "cloudposse/dynamic-subnets/aws"
+  version = "0.39.8"
+
+  availability_zones   = var.availability_zones
+  vpc_id               = module.vpc.vpc_id
+  igw_id               = module.vpc.igw_id
+  cidr_block           = module.vpc.vpc_cidr_block
+  nat_gateway_enabled  = true
+  nat_instance_enabled = false
+
+  context = module.this.context
+}
+
+module "mwaa" {
   source = "../.."
 
-  example = var.example
+  enabled                       = module.this.enabled
```

Comment:
```suggestion
  
```

no need to provide `enabled` since we are sending the entire context to the module

```
context = module.this.context
```

the context contains the `enabled `var

Reply:
@htplbc  please remove `enabled`

Reply:
Removed